Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/navigation/NavDrawerActionListener.java

```diff
@@ -0,0 +1,7 @@
+package com.novoda.todoapp.navigation;
+
+public interface NavDrawerActionListener {
```

Comment:
We probably don't want to leak the concept of nav drawer into core. This should be more domain oriented.